apps/docs/src/components/Dashboard/Billing.tsx

```diff
@@ -0,0 +1,173 @@
+import { useEffect, useState } from 'react'
+import Link from 'next/link'
+import { useUser } from '@/utils/useUser'
+import { Button } from '../Button'
+import { Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from '../ui/table'
+import SwitchToHobbyButton from '@/app/(docs)/docs/pricing/SwitchToHobbyButton'
+import SwitchToProButton from '@/app/(docs)/docs/pricing/SwitchToProButton'
```

Comment:
btw I think I need a bit of help on this one, I'm not really sure how the team + user + pricing tier are tied 🤔

Reply:
User can be part of multiple teams
Team has a tier, essentially teams is equivalent to e.g. GCP project. Team is paying for the usage

Reply:
Exactly so that part I got, but there's still a lot of code like `user.pricingTier.id` which is where I don't really follow.

Reply:
user.pricingTier is taken from `user.defaultTeam.pricingTier` if I remember correctly, there shouldn't be concept of pricing tier for the user, it can be part of multiple teams with different tiers